### DISCUSSION TITLE
How to install rails 7.0.1 on termux.

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#8855] [Show and tell]
Steps to install rails 7.0.1 :

1. pkg update && pkg upgrade 

2. pkg install ruby clang make binutils pkg-config libxslt

3. gem install nokogiri --platform=ruby -- --use-system-libraries 

![Screenshot_2022-01-31-22-35-35-754_com.brave.browser.jpg](https://user-images.githubusercontent.com/77871514/151839397-36fba093-eca0-4e99-ad3b-6183c260c04b.jpg)

### this will take some time to install. Be patient###

4. gem install rails 

###Now , to test if it works

5. rails new blog 

##now it will throw 'tzinfo-data gem not found' error.

6. cd blog && Open Gemfile in nano/vim/micro and comment it out like this . (Notice # after tzinfo-data )

![Screenshot_2022-01-31-22-33-10-563_com.termux.jpg](https://user-images.githubusercontent.com/77871514/151838998-1d868d43-af3d-4ae0-9c96-b242b8aad3a8.jpg)

7. Delete Gemfile.lock 

8. bundle install

9. rails server 

10. Voila !!!!

**Comment by spiderman-idog** [comment]
Thanks! It helped me setup jekyll on termux.
